content/en/blog/2024-05-16-modelix-platform-release-24.1/index.md

```diff
@@ -0,0 +1,58 @@
+---
+date: 2024-05-16
+title: "Modelix Platform Release 24.1"
+---
+
+Version **`24.1`** of the modelix platform is out now!
+
+The versions bundled in this release are:
+
+```
+modelixCoreVersion = 7.4.0
+incrementalVersion = 0.2.0
+buildToolsVersion = 1.5.0
+mpsPluginsVersion = 0.7.0
+```
+
+If you want to know how to use this platform in your Gradle build, check out the corresponding [documentation](https://docs.modelix.org/modelix/23.2/platform/howto/usage-platform.html).
+
+
+- [Feature Highlights](#feature-highlights)
+  - [Support for newer MPS Versions](#support-for-newer-mps-versions)
+  - [Stabilization of Bulk Model Synchronization Plugin](#stabilization-of-bulk-model-synchronization-plugin)
+- [Development Insights](#development-insights)
+  - [MPS Sync Plugin (experimental)](#mps-sync-plugin-experimental)
+  - [Adapter for LionWeb Bulk API (experimental)](#adapter-for-lionweb-bulk-api-experimental)
+  - [Modelix LinkedIn Page](#modelix-linkedin-page)
+
+# Feature Highlights
+
+## Support for newer MPS Versions
+
+Modelix components are now compatible with MPS 2023.2 and below.
```

Comment:
Maybe we could make this sentence mor explicit: `Modelix components are now compatible with MPS 2021.1 - 2023.2.`